elastic/integrations
# Add missing event fields

## What does this PR do?

This PR adds the missing event.* field mapping. Some of the mappings are already added ion this [PR](https://github.com/elastic/integrations/pull/5776/files) 

## Checklist



## How to test this PR locally

Build the integrations following the [guide](https://www.elastic.co/guide/en/integrations-developer/current/build-it.html) and check if the mapping is correct

## Related issues

- Closes #5869
